src/Elements/Geometry/Transform.cs

```diff
@@ -322,6 +322,27 @@ public Plane XZ()
         {
             return new Plane(this.Origin, this.YAxis);
         }
+        /// <summary>
+        /// Scale uniformly about the origin.
+        /// </summary>
+        /// <param name="amount">The amount to scale uniformly</param>
+        public void Scale(double amount)
```

Comment:
Although it'd be inconsistent with the original function, I'd still call this the factor rather than the amount to make it clear this is multiplicative.